api/konfig/plugins.go

```diff
@@ -77,7 +69,7 @@ type NotedFunc struct {
 // the home of kustomize plugins.
 func DefaultAbsPluginHome(fSys filesys.FileSystem) (string, error) {
 	return FirstDirThatExistsElseError(
-		"plugin home directory", fSys, []NotedFunc{
+		"plugin root", fSys, []NotedFunc{
```

Comment:
The "what" field wasn't actually being used before (fixed at https://github.com/kubernetes-sigs/kustomize/pull/3744/files#diff-d3f343eb9f99e66a4aa8533efdebf5b6ba1cd192319c8d783eddc7bc701350dcL26). This makes it so the message won't change after that fix.

Reply:
great, i added 'what' for debugging.